requirements/base.txt

```diff
@@ -150,11 +148,42 @@ numpy==1.26.4
     #   torchvision
     #   transformers
     #   unstructured
+nvidia-cublas-cu12==12.1.3.1
```

Comment:
These packages if I remember right show up when we pip compile on Linux rather than Mac. Can you rerun on a mac?